bezier/src/main/java/io/channel/bezier/compose/component/floating_icon_button/BezierFloatingIconButton.kt

```diff
@@ -0,0 +1,181 @@
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.channel.bezier.BezierIcons
+import io.channel.bezier.BezierTheme
+import io.channel.bezier.compose.component.avatar.BezierAvatar
+import io.channel.bezier.compose.component.floating_icon_button.properties.BezierFloatingIconButtonColor
+import io.channel.bezier.compose.component.floating_icon_button.properties.BezierFloatingIconButtonContent
+import io.channel.bezier.compose.component.floating_icon_button.properties.BezierFloatingIconButtonShape
+import io.channel.bezier.compose.component.floating_icon_button.properties.BezierFloatingIconButtonSize
+import io.channel.bezier.compose.component.floating_icon_button.properties.BezierFloatingIconButtonVariant
+import io.channel.bezier.compose.component.loader.BezierLoader
+import io.channel.bezier.compose.component.loader.properties.BezierLoaderSize
+import io.channel.bezier.compose.component.loader.properties.BezierLoaderVariant
+import io.channel.bezier.compose.foundation.ShadowStyle
+import io.channel.bezier.compose.foundation.bezierShadow
+import io.channel.bezier.extension.thenIf
+import io.channel.bezier.extension.toEmojiPainter
+import io.channel.bezier.icon.Plus
+
+@Composable
+fun BezierFloatingIconButton(
+        content: BezierFloatingIconButtonContent,
+        size: BezierFloatingIconButtonSize,
+        variant: BezierFloatingIconButtonVariant,
+        color: BezierFloatingIconButtonColor,
+        shape: BezierFloatingIconButtonShape,
+        onClick: () -> Unit,
+        modifier: Modifier = Modifier,
+        isLoading: Boolean = false,
+        enabled: Boolean = true,
+) {
+    val colorSchemes = color.getColorSchemes(variant)
+    val backgroundColor = colorSchemes.backgroundColor().color
+    val contentColor = colorSchemes.contentColor().color
+
+    val createdShape = shape.createShape(size)
+
+    Box(
+            modifier = modifier
+                    .bezierShadow(ShadowStyle.Shadow2, createdShape)
+                    .clip(createdShape)
+                    .thenIf(!enabled) {
+                        alpha(0.4f)
+                    }
+                    .background(backgroundColor)
+                    .clickable(enabled = enabled) { onClick() }
+                    .padding(size.containerPadding),
+            contentAlignment = Alignment.Center,
+    ) {
+        if (isLoading) {
+            BezierLoader(
+                    variant = BezierLoaderVariant.OnOverlay,
+                    size = BezierLoaderSize.Small,
+            )
+        }
+
+        Row(
```

Comment:
래퍼의 역할이니 Row보다 Box가 어떠신가유?
```suggestion
        Box(
```